package.json

```diff
@@ -5,12 +5,14 @@
   "version": "0.1.0",
   "author": "Kleros",
   "dependencies": {
-    "@kleros/archon": "^0.10.1",
+    "@kleros/archon": "^0.10.4",
```

Comment:
There is a problem with this version of Archon. Cases aren't loading with the upgraded version. Either downgrade back to Archon 0.10.2 (still reliant on scrypt), or else wait to merge this until web3 0.3.1 is released where the bug will be fixed

Reply:
What was the bug in Web3?

It looks like they already have an RC out, so we should wait.

Reply:
I just checked disputes do not load indeed.

I like the idea of giving information about products in a tooltip but I think now the icons are too tiny, the previous size was better. And we should use a darker color from the palette for tooltips, light pink on top of white is hard to read.

Reply:
It doesn't pin a version of util in web3-core-requestmanager but it depends on util > 0.12. Webpack uses an old version of util so when they are bundled together web3-core-requestmanager doesn't use the correct version and breaks. I saw the RC is out but that would take another week or so to be released most likely. Over the weekend I was working on migrating Archon to use Snowpack instead so that dep won't be there